app/models/draft_vaccination_record.rb

```diff
@@ -277,6 +278,7 @@ def writable_attribute_names
       programme_id
       session_id
       vaccine_id
+      first_active_wizard_step
```

Comment:
Ah just spotted this, I think we should remove this from `writable_attribute_names` because this a list of attributes which can be written to the `VaccinationRecord` model.